client/src/redux/reducers/index.js

```diff
@@ -1,33 +1,47 @@
 import { combineReducers } from 'redux';
 
 import addSourceWizardReducer from './addSourceWizardReducer';
+import confirmationModalReducer from './confirmationModalReducer';
 import credentialsReducer from './credentialsReducer';
 import factsReducer from './factsReducer';
 import reportsReducer from './reportsReducer';
-import sourcesReducer from './sourcesReducer';
 import scansReducer from './scansReducer';
+import sourcesReducer from './sourcesReducer';
+import statusReducer from './statusReducer';
 import toastNotificationsReducer from './toastNotificationsReducer';
-import confirmationModalReducer from './confirmationModalReducer';
-import viewOptionsReducer from './viewOptionsReducer';
 import userReducer from './userReducer';
-import statusReducer from './statusReducer';
+import viewOptionsReducer from './viewOptionsReducer';
 
 const reducers = {
   addSourceWizard: addSourceWizardReducer,
+  confirmationModal: confirmationModalReducer,
   credentials: credentialsReducer,
   facts: factsReducer,
   reports: reportsReducer,
-  sources: sourcesReducer,
   scans: scansReducer,
+  sources: sourcesReducer,
+  status: statusReducer,
   toastNotifications: toastNotificationsReducer,
-  confirmationModal: confirmationModalReducer,
-  viewOptions: viewOptionsReducer,
   user: userReducer,
-  status: statusReducer
+  viewOptions: viewOptionsReducer
 };
 
 const reduxReducers = combineReducers(reducers);
 
-export { reduxReducers, reducers };
+export {
+  reduxReducers,
+  reducers,
+  addSourceWizardReducer,
+  confirmationModalReducer,
+  credentialsReducer,
+  factsReducer,
+  reportsReducer,
+  scansReducer,
+  sourcesReducer,
+  statusReducer,
+  toastNotificationsReducer,
+  userReducer,
+  viewOptionsReducer
+};
```

Comment:
Is there a reason to export individual reducers?

Reply:
Looks like the only time it pops up is rolling through the unit testing. Did notice we cleaned it up once, I'll revert it back down to just exporting the `reduxReducers, reducers`... and follow it up with a similar refinement for the actions too